app/com/linkedin/drelephant/util/MemoryFormatUtils.java

```diff
@@ -94,7 +94,7 @@ public static long stringToBytes(String formattedString) {
       return 0L;
     }
 
-    Matcher matcher = REGEX_MATCHER.matcher(formattedString);
+    Matcher matcher = REGEX_MATCHER.matcher(formattedString.replace(",",""));
```

Comment:
Please add the unit test case to test the functionality of your code  MemoryFormatUtilsTest